x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/autoscaling/MlScalingReason.java

```diff
@@ -62,6 +71,7 @@ public MlScalingReason(StreamInput in) throws IOException {
                     String simpleReason) {
         this.waitingAnalyticsJobs = waitingAnalyticsJobs == null ? Collections.emptyList() : waitingAnalyticsJobs;
         this.waitingAnomalyJobs = waitingAnomalyJobs == null ? Collections.emptyList() : waitingAnomalyJobs;
+        this.waitingModels = waitingModels == null ? List.of() : waitingModels;
```

Comment:
In lines 72 and 73 `Collections.emptyList()` is being used instead of `List.of()`. Should `Collections.emptyList()` be used in line 74 too?

Reply:
@przemekwitek it doesn't matter. This code is staying in 8.0.0.